trestlebot/infra/entrypoints/create_ssp.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# create_new_ssp.py
+
+import argparse
+import sys
+
+from trestlebot.tasks.authored.ssp import AuthoredSSP, SSPIndex
+from trestlebot.cli import comma_sep_to_list
+from trestlebot.cli_base import EntrypointBase, handle_exception
+
+
+class CreateNewSSP(EntrypointBase):
```

Comment:
It might be helpful to discuss the overall approach in the description of this PR. The base entrypoint functionality is not being run, so these changes will not be committed to a branch.